.github/workflows/job_gpu_tests.yml

```diff
@@ -0,0 +1,134 @@
+name: GPU
+
+on:
+  workflow_call:
+    inputs:
+      test_type:
+        description: 'Type of tests to execute'
+        type: string
+        required: true
+      device:
+        description: 'Device name (igpu or dgpu)'
+        type: string
+        required: true
+      runner:
+        description: 'Machine on which the tests would run'
+        type: string
```

Comment:
I believe that this arg is more like `runner-labels`, I suggest tweaking the description to something like `Runner labels by which the runner will be chosen. Example "[ 'self-hosted', 'igpu' ]"`.